airflow_ext/gfw/models.py

```diff
@@ -90,5 +92,51 @@ def source_table_sensors(self, dag):
             for parts in self.source_table_parts(date=self.source_sensor_date_nodash())
         ]
 
+    def gcs_sensor(self, dag, bucket, prefix, date):
+        return GoogleCloudStoragePrefixSensor(
+            dag=dag,
+            task_id='source_exists_{}'.format(bucket),
+            bucket=bucket,
+            prefix='{}/{}'.format(prefix, date),
+            mode='reschedule',      # the sensor task frees the worker slot when the criteria is not yet met
+                                    # and it's rescheduled at a later time.
+            poke_interval=10 * 60,  # check every 10 minutes.
+            timeout=60 * 60 * 24    # timeout of 24 hours.
+        )
+
+    def source_path(self, date=None):
+        gcs_paths = self.config.get('source_paths') or self.config.get('source_path')
```

Comment:
As my comment below, I don't like mixing table paths and gcs path. I would keep `source_path` as it is and add `source_gcs_path`

Reply:
@enriquetuya I will work on the changes to have only `source_gcs_path`

Reply:
I think the same. In my opinion, it's better to separate in 2 fields.